src/components/SingleSelect/SingleSelect.less

```diff
@@ -0,0 +1,68 @@
+.lucid-SingleSelect {
+
+	.lucid-SingleSelect-Control {
+		.no-safari-flicker();
+		.box-sizing();
+		.gradient-animation( @featured-color-default-gradientStartColor, @featured-color-default-gradientEndColor);
+		cursor: pointer;
+		height: 28px;
```

Comment:
I think I've seen this height elsewhere, we might want to hoist it up into a variable. Something for another issue.